rest/data/version.go

```diff
@@ -160,7 +159,12 @@ func (vc *DBVersionConnector) GetVersionsAndVariants(skip, numVersionElements in
 			buildsInVersion := buildsByVersion[versionFromDB.Id]
 
 			// see if there are any active tasks in the version
-			versionActive := anyActiveTasks(buildsInVersion)
+			versionActive := false
```

Comment:
since versionActive isn't used until after the loop at 170, can it either be consolidated or moved to 191?
(edit: or actually maybe the logic at 194 should be moved before 170, because this build list isn't used in the case that the version is inactive)

Reply:
done